js/server/modules/@arangodb/foxx/router/validation.js

```diff
@@ -34,20 +33,18 @@ exports.validateParams = function validateParams (typeDefs, rawParams, type) {
     type = 'parameter';
   }
   const params = {};
-  for (const entry of typeDefs) {
-    const name = entry[0];
-    const def = entry[1];
-    if (def.schema && typeof def.schema.validate === 'function') {
-      const result = def.schema.validate(rawParams[name]);
-      if (result.error) {
-        const e = result.error;
-        e.message = e.message.replace(/^"value"/, `${type} "${name}"`);
-        throw e;
-      }
-      params[name] = result.value;
-    } else {
-      params[name] = rawParams[name];
+  for (const [name, def] of typeDefs) {
```

Comment:
Note that we no longer use the schemas directly. `check-args` always generates a `validate` method from the schema.